docker/database/Dockerfile

```diff
@@ -24,3 +24,5 @@ COPY --from=builder /azerothcore/env/dist/sql /sql
 # adding the "generate-databases.sh" to the directory "/docker-entrypoint-initdb.d"
 # because all scripts included in that directory will automatically be executed when the docker container starts
 COPY docker/database/generate-databases.sh /docker-entrypoint-initdb.d
+
+HEALTHCHECK --interval=5s --timeout=15s --start-period=30s --retries=3 CMD mysqladmin ping -h localhost
```

Comment:
Is that ninjaed in here? Those docker changes

Reply:
Sorry about that. The docker changes were from another PR that I used as a base. I've removed them from this PR.